resolver/contentResolver.go

```diff
@@ -0,0 +1,69 @@
+package resolver
+
+import (
+	"net/http"
+	"io/ioutil"
+	log "github.com/Sirupsen/logrus"
+	"encoding/json"
+	"github.com/Financial-Times/transactionid-utils-go"
+	"fmt"
+)
+
+type contentResolver struct {
+	tid string
```

Comment:
if we have the `tid` in this struct, this means I will need to create a new instance of the resolver for each request. I'd rather create the resolver only once and pass the `tid` as a parameter to the `ResolveContents` method.